SuperBuild/External_OpenCV.cmake

```diff
@@ -44,6 +44,11 @@ if(NOT DEFINED OpenCV_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
     list(APPEND ADDITIONAL_OPENCV_ARGS -DBUILD_PROTOBUF:BOOL=OFF)
   endif()
 
+  option(SlicerOpenCV_FORCE_RELEASE_PYTHON "Force the OpenCV build to use a release python" OFF)
+  if(SlicerOpenCV_FORCE_RELEASE_PYTHON)
+    list(APPEND ADDITIONAL_OPENCV_ARGS -DOPENCV_PYTHON_EXTRA_DEFINITIONS:STRING=CV_RELEASE_PYTHON # Force OpenCV to build using release python even if this is a Debug build
```

Comment:
Since the context of Slicer, we always build python in release, should this option always be on ?

Reply:
I wanted to make it an option so that this change only occurred when the developer wanted it to. Ie.: when doing separate debug/release builds with MSVC

Will change it to ON by default

Reply:
Since python is never built in Debug with Slicer, what do you think of always passing `-DCV_RELEASE_PYTHON:BOOL=ON`.